perses/tests/test_resume.py

```diff
@@ -64,6 +64,17 @@ def test_cli_resume():
             yaml.dump(y_doc, outfile)
         subprocess.run(["perses-relative", "test.yml"])
```

Comment:
Am I missing something, or does this only run perses _once_? We need to run it, stop it early (or extend it after it finishes), and then run it again.